src/routing/types.ts

```diff
@@ -8,7 +8,6 @@ import { FormikHelpers, FormikValues } from 'formik';
  */
 export interface RouterProps {
```

Comment:
Might not need to do this but should we rename this to `FormRouterProps` ?

Reply:
Ah good idea, I'll update.